includes/core/class-wc-payments-customer-service-api.php

```diff
@@ -0,0 +1,183 @@
+<?php
+/**
+ * Class file for WCPay\Core\WC_Payments_Customer_Service_API.
+ *
+ * @package WooCommerce Payments
+ */
+
+use WCPay\Database_Cache;
+use WCPay\Exceptions\API_Exception;
+use WCPay\Logger;
```

Comment:
I am not sure is this Logger class used somewhere. Do we need to remove it?

Reply:
Removed